Framework/PythonInterface/plugins/algorithms/EnggFitTOFFromPeaks.py

```diff
@@ -92,25 +93,23 @@ def _fit_dSpacingTOF(self, fitted_peaks_table):
             raise ValueError('Cannot fit a quadratic function with less than three peaks. Got a table of '
                              + 'peaks with ' + str(num_peaks) + ' peaks')
 
-        convert_tbl_alg = self.createChildAlgorithm('ConvertTableToMatrixWorkspace')
-        convert_tbl_alg.setProperty('InputWorkspace', fitted_peaks_table)
-        convert_tbl_alg.setProperty('ColumnX', 'dSpacing')
-        convert_tbl_alg.setProperty('ColumnY', 'X0')
-        convert_tbl_alg.execute()
-        d_tof_conversion_ws = convert_tbl_alg.getProperty('OutputWorkspace').value
+        d_tof_conversion_ws = ConvertTableToMatrixWorkspace(
+            InputWorkspace=fitted_peaks_table,
+            ColumnX='dSpacing',
+            ColumnY='X0'
+        )
```

Comment:
This also leaves workspaces in the ADS, which is probably not intended. Adding` StoreInADS=False` should restore previous behaviour.